projects/1.20/assets/stoneworks/stoneworks/lang/zh_cn.json

```
@@ -0,0 +1,584 @@
+  "block.stoneworks.basalt_brick_wall": "玄武岩砖墙",
+  "block.stoneworks.basalt_bricks": "玄武岩砖",
+  "block.stoneworks.basalt_paver_slab": "玄武岩地砖台阶",
+  "block.stoneworks.basalt_paver_stairs": "玄武岩地砖楼梯",
+  "block.stoneworks.basalt_paver_wall": "玄武岩地砖墙",
+  "block.stoneworks.basalt_pavers": "玄武岩地砖",
+  "block.stoneworks.basalt_plate_slab": "切制玄武岩台阶",
+  "block.stoneworks.basalt_plate_stairs": "切制玄武岩楼梯",
+  "block.stoneworks.basalt_plate_wall": "切制玄武岩墙",
+  "block.stoneworks.basalt_plates": "切制玄武岩台阶",
+  "block.stoneworks.basalt_shingle_slab": "瓦制玄武岩台阶",
+  "block.stoneworks.basalt_shingle_stairs": "瓦制玄武岩楼梯",
+  "block.stoneworks.basalt_shingle_wall": "瓦制玄武岩墙",
+  "block.stoneworks.basalt_shingles": "瓦制玄武岩",
+  "block.stoneworks.basalt_tile_slab": "竹制玄武岩台阶",
+  "block.stoneworks.basalt_tile_stairs": "竹制玄武岩楼梯",
+  "block.stoneworks.basalt_tile_wall": "竹制玄武岩墙",
+  "block.stoneworks.basalt_tiles": "竹制玄武岩",
+  "block.stoneworks.blackstone_paver_slab": "黑石地砖台阶",
+  "block.stoneworks.blackstone_paver_stairs": "黑石地砖楼梯",
+  "block.stoneworks.blackstone_paver_wall": "黑石地砖墙",
+  "block.stoneworks.blackstone_pavers": "黑石地砖",
+  "block.stoneworks.blackstone_pillar": "黑石柱",
+  "block.stoneworks.blackstone_plate_slab": "切制黑石台阶",
+  "block.stoneworks.blackstone_plate_stairs": "切制黑石楼梯",
+  "block.stoneworks.blackstone_plate_wall": "切制黑石墙",
+  "block.stoneworks.blackstone_plates": "切制黑石",
+  "block.stoneworks.blackstone_shingle_slab": "瓦制黑石台阶",
+  "block.stoneworks.blackstone_shingle_stairs": "瓦制黑石楼梯",
+  "block.stoneworks.blackstone_shingle_wall": "瓦制黑石墙",
+  "block.stoneworks.blackstone_shingles": "瓦制黑石",
+  "block.stoneworks.blackstone_tile_slab": "竹制黑石台阶",
+  "block.stoneworks.blackstone_tile_stairs": "竹制黑石楼梯",
+  "block.stoneworks.blackstone_tile_wall": "竹制黑石墙",
+  "block.stoneworks.blackstone_tiles": "竹制黑石",
+  "block.stoneworks.calcite_brick_slab": "方解石砖台阶",
+  "block.stoneworks.calcite_brick_stairs": "方解石砖楼梯",
+  "block.stoneworks.calcite_brick_wall": "方解石砖墙",
+  "block.stoneworks.calcite_bricks": "方解石砖",
+  "block.stoneworks.calcite_paver_slab": "方解石地砖台阶",
+  "block.stoneworks.calcite_paver_stairs": "方解石楼梯",
+  "block.stoneworks.calcite_paver_wall": "方解石地砖墙",
+  "block.stoneworks.calcite_pavers": "方解石地砖",
+  "block.stoneworks.calcite_pillar": "方解石柱",
+  "block.stoneworks.calcite_plate_slab": "切制方解石台阶",
+  "block.stoneworks.calcite_plate_stairs": "切制方解石楼梯",
+  "block.stoneworks.calcite_plate_wall": "切制方解石墙",
+  "block.stoneworks.calcite_plates": "切制方解石",
+  "block.stoneworks.calcite_shingle_slab": "瓦制方解石台阶",
+  "block.stoneworks.calcite_shingle_stairs": "瓦制方解石楼梯",
+  "block.stoneworks.calcite_shingle_wall": "瓦制方解石墙",
+  "block.stoneworks.calcite_shingles": "瓦制方解石",
+  "block.stoneworks.calcite_tile_slab": "竹制方解石台阶",
+  "block.stoneworks.calcite_tile_stairs": "竹制方解石楼梯",
+  "block.stoneworks.calcite_tile_wall": "竹制方解石墙",
+  "block.stoneworks.calcite_tiles": "竹制方解石",
+  "block.stoneworks.chiseled_andesite": "錾制安山岩",
```

```suggestion
  "block.stoneworks.chiseled_andesite": "雕纹安山岩",
```
自1.19.3版本起原版的`錾制`改为`雕纹`，请注意对1.20和1.20Fabric的文件进行修改
1.19版本的文件可随意挑一个小版本，1.19.2和1.19.4均可，如果选择1.19.4也请进行修改